### PR TITLE
[16.0][FIX] stock_picking_line_sequence

### PR DESCRIPTION
**Pull request by AungKoKoLin1997**
This PR is to follow up the updating behavior of the odoo standard done by this PR(https://github.com/odoo/odoo/pull/144276).

@qrtl

### CODE REVIEW

File: stock_picking_line_sequence/models/stock.py [hunk 1]
```diff
@@ -34,25 +34,27 @@ def create(self, values):
...
+        uom = move.product_uom or move_line.product_uom_id
+        name = move.product_id.display_name
+        description = move.description_picking
+        if description == name or description == move.product_id.name:
+            description = False
+        product = move.product_id
+        line_key = (
+            f"{product.id}_{product.display_name}_" f'{description or ""}_{uom.id}'
+        )
+        aggregated_properties["line_key"] = line_key
```
Comment (yostashiro) [inline stock_picking_line_sequence/models/stock.py]: @AungKoKoLin1997 As we just talked, can you please check and add a docstring to this method to explain why this override is necessary.

Comment (AungKoKoLin1997) [reply stock_picking_line_sequence/models/stock.py]: @yostashiro  I misunderstood about it. We can get rid of extending this method.

**Comment by yostashiro** [comment]
@AungKoKoLin1997 Please squash commits and add more to the commit message, explaining that the current implementation breaks if used together with the mrp module.

**Review by yostashiro — APPROVED** [review]
Code review.

**Comment by AungKoKoLin1997** [comment]
@oca/stock-logistics-workflow-maintainers
Can you please review this PR?

**Review by rousseldenis — APPROVED** [review]
Great to remove such code!

**Comment by rousseldenis** [comment]
@AungKoKoLin1997 Is this ready ?

**Comment by AungKoKoLin1997** [comment]
> Is this ready ?

@rousseldenis Yes, it is.

**Pull request merged**